solar_installation/data/website_page.xml

```diff
@@ -6,4 +6,10 @@
         <field name="url">/get-a-quote</field>
         <field name="website_id" ref="website.default_website"/>
     </record>
+    <record id="unpublish_contact_us_page" model="website.page">
+        <field name="is_published" eval="False"/>
```

Comment:
This doesn't seem to be sufficient, as the button is still there, leading to a 404.